guess_the_number.py

```diff
@@ -34,17 +34,22 @@ def main():
 
     (low, high) = configure_range()
     secret = generate_secret(low, high)
+    total_guess = 0
 
     while True:
         guess = get_guess()
         result = check_guess(guess, secret)
         print(result)
 
+        if result != correct:
+            total_guess = total_guess + 1
```

Comment:
instead of using two if statements, use the check_guess function

Reply:
could make total_guess a global variable and +1 to total variable as a line in the check_guess function